OCPBUGS#27913: Doc improvements related to "Scaling up storage of clusters" (#71942)
openshift/openshift-docs: storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc

* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#about-lvmcluster_logical-volume-manager-storage[About the LVMCluster custom resource]

include::modules/lvms-scaling-storage-of-clusters-using-rhacm.adoc[leveloffset=+2]

[role="_additional-resources"]
.Additional resources

* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-unsupported-devices_logical-volume-manager-storage[Devices not supported by {lvms}]

* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-integrating-software-raid-arrays_logical-volume-manager-storage[Integrating software RAID arrays with {lvms}]

* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#about-lvmcluster_logical-volume-manager-storage[About the LVMCluster custom resource]

// Expanding PVCs
include::modules/lvms-scaling-storage-expand-pvc.adoc[leveloffset=+1]

[role="_additional-resources"]
.Additional resources

> * xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-scaling-storage-of-clusters-using-web-console_logical-volume-manager-storage[Scaling up storage by adding capacity to your cluster]

Reviewer: Link text is different here:
```suggestion
* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-scaling-storage-of-clusters-using-web-console_logical-volume-manager-storage[Scaling up the storage of clusters by using the web console]
```
(remember to remove "by" here if you accept my suggestion to remove it from the module heading)

Author: Removed these references and add the following reference:

* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-about-scaling-storage-of-cluster_logical-volume-manager-storage[Scaling up the storage of clusters]

As this section includes all three ways of scaling up the storage.

> * xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-scaling-storage-of-clusters-using-rhacm_logical-volume-manager-storage[Scaling up storage by adding capacity to your cluster using {rh-rhacm}]

Reviewer: Link text is different here:
```suggestion
* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-scaling-storage-of-clusters-using-rhacm_logical-volume-manager-storage[Scaling up the storage of clusters by using {rh-rhacm}]
```
(remember to remove "by" here if you accept my suggestion to remove it from the module heading)

Author: Removed these references and add the following reference:

* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-about-scaling-storage-of-cluster_logical-volume-manager-storage[Scaling up the storage of clusters]

As this section includes all three ways of scaling up the storage.
